packages/circuit-ui/components/NotificationModal/NotificationModal.spec.tsx

```diff
@@ -50,6 +50,12 @@ describe('NotificationModal', () => {
       expect(baseElement).toMatchSnapshot();
     });
 
+    it('should render without an image', () => {
+      const { image, ...props } = baseNotificationModal;
+      const { baseElement } = renderNotificationModal(props);
+      expect(baseElement).toMatchSnapshot();
+    });
+
```

Comment:
Should we also add an example in the Storybook too?

Reply:
I don't think there's much value in it. We're already over our Chromatic limit for this month, so I'm trying not to increase the number of stories too much. 

Certainly not ideal to have to limit ourselves this way. We (aka @robinmetral 🙈) should look into a solution for this (perhaps we can combine some stories?).